app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -51,11 +51,25 @@ class PatientSearchResultItemView(
     renderPatientNameAgeAndGender(searchQuery, model)
     renderPatientAddress(model.address)
     renderPatientDateOfBirth(model.dateOfBirth)
-    renderPatientPhoneNumber(model.phoneNumber)
+    val number = highlightPatientPhoneNumber(searchQuery, model)
+    renderPatientPhoneNumber(number)
     renderVisited(model.lastSeen)
     renderLastSeen(model.lastSeen, currentFacilityId)
   }
 
+  private fun highlightPatientPhoneNumber(searchQuery: String?, patientSearchResult: PatientSearchResultViewModel): PhoneNumber {
+    val canHighlightNumber = searchQuery?.let { searchQuery.isNotBlank() && patientSearchResult.phoneNumber.toString().contains(it) } == true
+    return if (canHighlightNumber) {
+      PhoneNumber.Highlighted(
+          patientNumber = patientSearchResult.phoneNumber.toString(),
+          highlightStart = patientSearchResult.phoneNumber.toString().indexOf(searchQuery!!, ignoreCase = true),
```

Comment:
We can extract out the index to a variable since it's being used for both `highlightStart` & `highlightEnd`